docs/publishing/advanced_publishing_options.md

```diff
@@ -1,14 +1,14 @@
 # Advanced publishing options
 
-There are three ways in which you can schedule content to be published in the future:
+There are three ways in which you can schedule content to be revealed or hidden in the future:
```

Comment:
But it's not just reveal or hide, is it? "Publishing" and "revealing" are not the same actions - may not be easy to explain to editors, but we should keep the distinction somehow.

Ok, Content Scheduler block actually uses already published Content items (IIRC), so the previous version was misleading as well.